inc/tasks/promotions.php

```diff
@@ -167,7 +167,7 @@ function task_promotions($task)
 			$plugins->run_hooks('task_promotions', $args);
 		}
 
-		$query2 = $db->simple_select("users", "uid,{$usergroup_select}", $sql_where);
+		$query2 = $db->simple_select("users", "uid,usergroup,additionalgroups,displaygroup,{$usergroup_select}", $sql_where);
```

Comment:
`$usergroup_select` might seem redundant now, but it is passed by references in the plugin hook above, so the best would be to keep for now.